.github/workflows/e2e-tests.yaml

```
@@ -48,6 +48,25 @@ jobs:
 
+  e2e-tests:
+    runs-on: ubuntu-latest
+    needs: setup-cluster
+    steps:
+        - name: Checkout code
+          uses: actions/checkout@v4
+          with:
+            submodules: recursive
+
+        - name: Set up Go
+          uses: actions/setup-go@v3
```

actionlint (1.7.7)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)